### ISSUE TITLE
Continue Watching not loading next episode when using Skip button (after completing an episode)

### ISSUE THREAD

**Issue by adrian-vapor** [Stremio/stremio-bugs#890]
**Describe the bug**
The issue only occurs if you use the Skip button either using your TV's remote or via the on-screen menu. If you instead use the Seek (Fast Forward) button on your TV remote, or click on the "Skip CREDITS" button (which appears on-screen near the end of an episode) to navigate to the end of the episode, you can back out of that series without having to watch a few seconds of the next episode, and the series remains in the Continue Watching section.

**To Reproduce**
1. Go to any TV series
2. Use your TV remote's Skip button or the on-screen menu Skip button to navigate to the end of the episode
3. The next episode is not added to the Continue Watching section

**Expected behavior**
Using the Skip button to get to the next episode quickly should retain that series in my Continue Watching list. It was working correctly until this software release - https://blog.stremio.com/stremio-tech-update-31-stremio-android-tv-updated/

**TV**
 - Device: Sony BRAVIA 4K UR3 (A8H)
 - OS: Android TV v14
 - Stremio: v1.6.11

**Comment by TRtomasz** [comment]
Skip button starts the next episode not add it to the continue watching if you click skip and then exit immediately this is expected

**Comment by adrian-vapor** [comment]
Fair enough, but for many months, maybe even a year I was using the Skip button on my remote at the end of episodes to tee up the next episode quickly in Continue Watching, for my next viewing session...so you're saying when this worked like this for so long, before the latest software update, it was actually a bug? Can you not just make it work like it used to...as it is the easiest and fastest way to get to the next episode after you've completed watching an episode...especially Netflix shows which have a lot of post credits. Thanks

**Comment by JonDotExe** [comment]
> Fair enough, but for many months, maybe even a year I was using the Skip button on my remote at the end of episodes to tee up the next episode quickly in Continue Watching, for my next viewing session...so you're saying when this worked like this for so long, before the latest software update, it was actually a bug? Can you not just make it work like it used to...as it is the easiest and fastest way to get to the next episode after you've completed watching an episode...especially Netflix shows which have a lot of post credits. Thanks

I agree. This is how it 'should' work. 

'Fixing' this this behavior now looks like a bug.
